### ISSUE TITLE
Can't get accurate column/row separations

### ISSUE THREAD

**Issue by LiamVisionary** [hako-mikan/sd-webui-regional-prompter#323]
**Describe the bug**
For the life of me, I can't figure out why I can't get a simple separation set up.

Here is my prompt (without LORA's for simplified testing):
```
score_9, score_8_up, score_7_up, source_anime, girl looking curiously at a guy, source anime ADDCOMM
haruno_sakura  ADDROW
uchiha_sasuke 
```

And my setup is:
Use Common Prompt: Enabled
Main Splitting: Rows
Divide Ratio: 1,1

Which appears as such:
![Screenshot 2024-05-01 at 2 24 48 AM](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/102138294/00b7a2e1-20a9-47ed-9c79-f72fa9f93588)

So to my understanding, that should result in haruno_sakura in the top row and uchiha_sasuke in the bottom row.

However here is the result:
![Screenshot 2024-05-01 at 2 25 55 AM](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/102138294/e55d8e82-7ec8-4d30-b8cf-c274b590785c)
and
![Screenshot 2024-05-01 at 2 26 43 AM](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/102138294/dfabdf06-fad5-4280-bf57-b8485c57269f)
and
![Screenshot 2024-05-01 at 2 27 34 AM](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/102138294/56693c1c-99e5-4833-8d9c-91b3a7dd8c7e)

**Environment**
Web-UI version: 1.9.0

SD Version:
Pony Diffusion V6 XL

**Other Enabled Extensions**
Config-Presets
adetailer
sd-webui-traintrain
sd_dreambooth_extension

**Comment by LoRAMilkshake** [comment]
can you try making two columns instead and share your results?

**Comment by LiamVisionary** [comment]
@LoRAMilkshake Sure thing, here it is:
![image](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/102138294/cc728769-7598-4c72-b9ce-2b9a2c156e19)
![image](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/102138294/84804ca7-16b8-426b-ae03-c13ca763438f)
![image](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/102138294/58eb1fd2-bc25-4ad0-bcfc-1867189bc4cd)

Ran it 3 times at 512x512. Here is once at 1024x1024
![image](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/102138294/0b6b4417-a87b-4a14-a874-6cc9083de075)

Very strange...

**Comment by LoRAMilkshake** [comment]
yeah, I'm having troubles making it work with PDXL as well, I think is not supported, would be nice if someone else chip in to confirm

**Comment by LiamVisionary** [comment]
@LoRAMilkshake I'm glad I'm not the only once. I also just completely deleted my webui, reinstalled it, copied over the same models and embeddings, and the issue still persists. No additional extensions

**Comment by rcragun** [comment]
I have been unable to get layouts that match my specifications except very rarely. Examples:

# Regional prompter deactivated

```
Harry Potter, 
Hermione Granger
```
![grid-0012](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/5415664/dcf5f34f-3f4d-449b-a8f2-d2d4b7559abd)

# Column split

* Split: `1,1`, 
* Base prompt: off
* Common prompt: off

```
Harry Potter BREAK
Hermione Granger
```
or
```
Harry Potter ADDCOL
Hermione Granger
```
![grid-0008](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/5415664/db35c64e-8e8d-4d33-9a44-9da9bf21a3e6)
![grid-0009](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/5415664/54110baf-acdd-4348-9788-50533f4ae0e1)
![grid-0010](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/5415664/b2aad3ef-18fc-4747-a414-d2ffd246aab6)

# Row split

* Split: `1,1`, 
* Base prompt: off
* Common prompt: off

```
Harry Potter BREAK
Hermione Granger
```
or
```
Harry Potter ADDROW
Hermione Granger
```
![grid-0014](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/5415664/f367c8d0-7b35-4245-97c2-c414bf539782)

**Comment by SheLikesCloth** [comment]
I have also never been able to get this to work. I tried this very simple prompt with a common prompt, main split as “Columns”, and divide ratio at “1,1”:
```
portrait of a man and a woman ADDCOMM
man ADDCOL
woman
```
But the order of the people in the image is no better than if I don’t use Regional Prompter (and just have the words “man” and “woman” in the desired order from left to right).

I also tried 
- splitting by row
- both attention and latent modes
- five (very) different models
- generating in batches and not in batches (because of issue https://github.com/hako-mikan/sd-webui-regional-prompter/issues/296). 
- using `ADDCOMM`, `ADDCOL`, and `ADDROW` or with just `BREAK`

I have never been able to produce evidence that Regional Prompter is having any effect.

**Comment by hako-mikan** [comment]
I can’t figure anything out with just that, so please follow the guidance displayed when creating an issue and include details like the execution environment.

**Comment by hako-mikan** [comment]
If you are using reForge, fixed.

**Comment by hako-mikan** [comment]
@LiamVisionary
It is difficult to divide the screen vertically and depict multiple people. 
This is because the model has not been trained on such images. 
Placing a woman at the top may be somewhat feasible, but arranging another person below is challenging. 
In my own attempts with your prompt, vertical division did not work well.
If you want to achieve this type of composition, please use it in combination with ControlNet. 
To check whether the extension is functioning correctly, try the sample provided in the ReadMe.
